cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/executor/DatasetAdminOpHTTPHandler.java

```diff
@@ -39,20 +40,19 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import javax.ws.rs.GET;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 
 /**
- * Provides REST endpoints for {@link DatasetAdmin} operations.
+ * V3 Dataset Admin Ops Handler
```

Comment:
Leave the comment on this class (since this is what will end up staying), and add "V2 Dataset Admin..." to the other class.